bukuserver/views.py

```diff
@@ -482,9 +481,14 @@ def create_model(self, form):
 
 class StatisticView(BaseView):  # pylint: disable=too-few-public-methods
 
+    def __init__(self, *args, **kwargs):
+        self.bukudb = args[0]
```

Comment:
with using kwargs method above, it will be `self.bukudb = kwargs.pop('bukudb', None)`

and on `index` method use `self.bukudb if self.bukudb else BukuDb()`

Reply:
see the remark above on `args` vs `kwargs`